### DISCUSSION TITLE
Raw link to PDF files should display and not force download

### DISCUSSION THREAD

**Discussion by albertz** [community/community#48074] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

In most cases, you just want to display the PDF. You often still want to do that instead of using the GitHub built-in PDF preview because many plugins (e.g. Zotero) cannot handle the PDF properly. And also, the browser PDF viewer is more featurefull compared to the GitHub PDF preview. And also, it directly shows all pages, unlike the GitHub PDF preview which just shows the first 5 pages.

As I understand, you just need to remove the `content-disposition: attachment`.

This is a frequent request by others as well: https://webapps.stackexchange.com/questions/48061/can-i-trick-github-into-displaying-the-pdf-in-the-browser-instead-of-downloading

**Comment by haxfn** [comment]
Is there any way of doing this from github instead other than github-pages? I need to use the browser pdf viewer! I don't want to explicitly download and then open the pdf.

https://USERNAME.github.io/REPONAME/PATH.pdf
https://cdn.jsdelivr.net/gh/USERNAME/REPONAME@BRANCHNAME/PATH.pdf

both of these solutions are there, but

I hate github more for not having this:

https://raw.githubusercontent.com/USERNAME/REPONAME/refs/heads/BRANCHNAME/PATH.pdf
